modules/ossm-threescale-applying-external-service-entry-objects.adoc

```diff
@@ -49,25 +96,52 @@ spec:
   location: MESH_EXTERNAL
   resolution: DNS
 ----
+.. Add the `DestinationRule` CR to a file called _destination-rule-threescale-saas-system.yml:_
 +
-You can use the `oc apply` command with either of the following methods to apply the objects:
+.DestinationRule CR
+[source,terminal]
+----
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: destination-rule-threescale-saas-system
+spec:
+  host: multitenant.3scale.net
+  trafficPolicy:
+    tls:
+      mode: SIMPLE
+      sni: multitenant.3scale.net
+----
+
+. Apply and save the external `ServiceEntry` CR for the 3scale Hosted system to your cluster, by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <service-entry-threescale-saas-system.yml>
+----
 
-** Save the objects to one or more files, and then use the following syntax:
+.. Alternatively, apply the external `ServiceEntry` CR for the 3scale Hosted system to your cluster without first saving the file, by running the following command:
```

Comment:
This command as far as I can tell would require a previously saved `service-entry-threescale-saas-system.yml` file and it just applies the configuration in much the same way as the preceding command. You could remove this alternative step entirely to minimise the procedure.